client/lib/pages/add-plant.dart

```diff
@@ -34,190 +219,377 @@ class _AddPlant extends State<AddPlant> {
-                                keyboardType: TextInputType.number,
-                                validator: (String value) {
-                                  int temp = int.tryParse(value);
-
-                                  if (temp == null) {
-                                    return 'Moet een getal zijn.';
-                                  }
-
-                                  if (maxTemp != null && maxTemp < temp) {
-                                    return 'Mag niet hoger zijn\n dan max temp.';
-                                  }
-
-                                  minTemp = temp;
-                                  return null;
-                                },
-                                onSaved: (String value) {
-                                  newPlant.minTemp = minTemp;
-                                },
-                            )
+                            child: newPlant.imageName != null &&
+                                File(newPlant.imageName).existsSync()
+                                ? Image.file(
+                                    File(newPlant.imageName),
+                                    width: 150,
+                                    height: 150,
+                                  )
+                                : Image.asset("assets/images/image-placeholder.png")
                           ),
                           Expanded(
-                            flex: 4,
-                            child: AddPlantTextField(
-                                label: 'Max temperatuur',
-                                keyboardType: TextInputType.number,
-                                validator: (String value) {
-                                  int temp = int.tryParse(value);
-
-                                  if (temp == null) {
-                                    return 'NaN';
-                                  }
-
-                                  if (minTemp != null && minTemp > temp) {
-                                    return 'Mag niet lager zijn\n dan min temp.';
-                                  }
-
-                                  maxTemp = temp;
-                                  return null;
-                                },
-                                onSaved: (String value) {
-                                  newPlant.maxTemp = maxTemp;
-                                },
-                            )
+                            flex: 6,
+                            child: FloatingActionButton(
+                              child: Icon(Icons.image),
+                              onPressed: () => showDialog(
+                                  context: context,
+                                  builder: (BuildContext context) {
+                                    return AlertDialog(
+                                        title: Text("Maak een keuze"),
+                                        content: SingleChildScrollView(
```

Comment:
The `SingleChildScroll` view should be moved to higher up in the hierarchy. It should be the parent widget of the upper most padding widget to prevent the scrolling looking like this:
![2020-06-03 20_50_22-ONEPLUS A6003](https://user-images.githubusercontent.com/22012835/83677350-5410d980-a5dc-11ea-9d1b-d47037fab4b8.png)